### PR TITLE
Add powershell script to install dir for fixing CH340 driver issues

### PR DESCRIPTION
**Pull request by kounocom**
Adds a powershell script to the installation directory if CH340 drivers are selected. The script uninstalls any ch340 driver, replaces it with a known working one, and blocks all future updates to that driver. Useful for some D1 minis which have counterfeit CH340s, which do not work with the latest 2023 driver.

Note: the script currently downloads the driver from my own CDN, which is probably not the safest option. can change tho.

### CODE REVIEW

File: windows/web/fix_ch340.ps1 [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# AFAIK this is the only HWID that counterfeit CH340s use
+# Well, they have 2 HWIDs according to Windows, but only 1 is needed for detection
+$HardwareID = "USB\VID_1A86&PID_7523"
+
+# ! Currently it gets the driver from my own CDN! This might be a security problem.
+$DriverZipUrl = 'cdn.kouno.xyz/YZSvN1b5.zip'
```
Comment (ImUrX) [inline windows/web/fix_ch340.ps1:6]: i agree we need to change this, I suggest uploading it to Github and use their CDN

**Review by ImUrX — CHANGES_REQUESTED** [review]

File: windows/web/fix_ch340.ps1 (hunk 1 above)
Comment (ImUrX) [reply windows/web/fix_ch340.ps1:6]: Or just use a bundled file lol

**Comment by kounocom** [comment]
no more downloading stuff

**Comment by kounocom** [comment]
after a few days of everything working, the driver stopped registering to my CH340 because of the registry changes, needs more testing

**Pull request closed without merging**